pkg/dash/watcher.go

```diff
@@ -132,7 +132,7 @@ func (cw *ConfigWatcher) Watch(ctx context.Context) {
 			done = true
 			logger.Infof("shutting down config watcher")
 		case _ = <-cw.FileWatcher.Events():
-			if err := cw.watcherConfig.UseContext(ctx, cw.watcherConfig.ContextName()); err != nil {
+			if err := cw.watcherConfig.UseContext(ctx, cw.watcherConfig.CurrentContext()); err != nil {
```

Comment:
A potentially interesting edge case (doesn't have to be addressed here) is what should happen if a config is loaded but becomes invalid (e.g. deleted) while Octant is running.